### ISSUE TITLE
(None, None, None) shape of y_true tensor in some cases

### ISSUE THREAD

**Issue by leokster** [leokster/CVAE#4]
It can happen that the tensor shape of y_true has not been evaluated for the validation set and since is (None, ..., None). 
See https://stackoverflow.com/questions/61871736/shape-none-none-of-ground-truth-labels-when-using-custom-keras-data-generator as reference. The problem can be solved by uncommenting the following line

https://github.com/leokster/CVAE/blob/cde6ee188a97c124ac3e44dda768e12a39115e08/variational_autoencoder/models.py#L149
